new-lamassu-admin/src/pages/Triggers/Triggers.js

```diff
@@ -74,72 +78,96 @@ const Triggers = () => {
 
   return (
     <>
-      <TitleSection title="Compliance Triggers" className={classes.tableWidth}>
-        <Box display="flex" alignItems="center">
+      <TitleSection
+        title="Compliance Triggers"
+        button={{
+          text: 'Advanced settings',
+          icon: ListingViewIcon,
+          inverseIcon: ReverseListingViewIcon,
+          toggle: setAdvancedSettings
+        }}
+        className={classes.tableWidth}>
+        {!advancedSettings && (
+          <Box display="flex" alignItems="center">
+            <Box
+              display="flex"
+              alignItems="center"
+              justifyContent="end"
+              mr="-5px">
+              <P>Reject reused addresses</P>
+              <Switch
+                checked={rejectAddressReuse}
+                onChange={event => {
+                  addressReuseSave({ rejectAddressReuse: event.target.checked })
+                }}
+                value={rejectAddressReuse}
+              />
+              <Label2 className={classes.switchLabel}>
+                {rejectAddressReuse ? 'On' : 'Off'}
+              </Label2>
+              <Tooltip width={304}>
+                <P>
+                  This option requires a user to scan a different cryptocurrency
+                  address if they attempt to scan one that had been previously
+                  used for a transaction in your network
+                </P>
+              </Tooltip>
+            </Box>
+          </Box>
+        )}
+      </TitleSection>
+      {!advancedSettings && (
+        <>
           <Box
+            marginBottom={2}
+            className={classes.tableWidth}
             display="flex"
-            alignItems="center"
-            justifyContent="end"
-            mr="-5px">
-            <P>Reject reused addresses</P>
-            <Switch
-              checked={rejectAddressReuse}
-              onChange={event => {
-                addressReuseSave({ rejectAddressReuse: event.target.checked })
-              }}
-              value={rejectAddressReuse}
-            />
-            <Label2 className={classes.switchLabel}>
-              {rejectAddressReuse ? 'On' : 'Off'}
-            </Label2>
-            <Tooltip width={304}>
-              <P>
-                This option requires a user to scan a different cryptocurrency
-                address if they attempt to scan one that had been previously
-                used for a transaction in your network
-              </P>
-            </Tooltip>
+            justifyContent="flex-end">
```

Comment:
With the `className` defined, couldn't you add the `display="flex"` and `justifyContent="flex-end"` to `classes.tableWidth` and give it a more relevant name?

Reply:
Just noticed this section of the code is not new. You can choose to ignore this comment